src/mad_spark_alt/core/qadi_prompts.py

```
@@ -0,0 +1,187 @@
+        "top_p": 0.9
+    }
+}
+
+
+# Unified evaluation criteria weights
+EVALUATION_CRITERIA = {
+    "novelty": 0.2,      # How innovative/unique
+    "impact": 0.3,       # Level of positive change
+    "cost": 0.2,         # Resource efficiency (inverse - lower cost = higher score)
+    "feasibility": 0.2,  # Implementation practicality
+    "risks": 0.1         # Risk mitigation (inverse - lower risk = higher score)
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)